moveit_ros/planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -155,44 +193,35 @@ class MoveGroup::MoveGroupImpl
   }
 
   template<typename T>
-  void waitForAction(const T &action, const ros::Duration &wait_for_server, const std::string &name)
+  void waitForAction(const T &action, const std::string &name, const ros::WallTime &timeout, double allotted_time)
```

Comment:
i agree time vars should be grouped. since its private you're right, it doesn't matter